packages/shared/components/Profile.svelte

```diff
@@ -48,7 +48,7 @@
             </div>
             <NetworkIconBadge {networkType} {networkProtocol} />
         </div>
-        <div class="flex flex-row items-baseline space-x-1.5 mb-2">
+        <div class="flex flex-row items-baseline space-x-1.5 mb-2 w-full">
```

Comment:
with this, short names are not centered anymore. Try `content-center`

Reply:
`justify-center` seems to do the trick

Reply:
pushed the fix